src/Language/Mimsa/Actions.hs

```diff
@@ -20,16 +19,21 @@ import qualified Data.Set as S
 import Data.Text (Text)
 import Language.Mimsa.Parser (parseExprAndFormatError)
 import Language.Mimsa.Project
-  ( getCurrentBindings,
-    getCurrentTypeBindings,
-  )
 import Language.Mimsa.Store
   ( createStoreExpression,
     substitute,
   )
-import Language.Mimsa.Store.ExtractTypes
 import Language.Mimsa.Typechecker
-import Language.Mimsa.Types
+import Language.Mimsa.Types.AST
```

Comment:
This is the cost of removing the big types file, however it does say to me more clearly that this file is doing too much.